commands.js

```diff
@@ -410,6 +410,22 @@ let captureDom = (win) => {
         cy.writeFile(`./${vtConfFile.debug}/${imageName}-${imageType}/${imageName}.json`, dom);
     }
 };
+let scrollToTop = (win) =>{
+    let tries = 0;
+    let scrollOffset = 1;
+    do {
+        // Translate to the top of the page and then capture the dom
+        win.eval(`document.body.style.transform="translateY(0)"`);
```

Comment:
This should be more of a check scroll vs "scrollToTop"

Reply:
So why not copy the functionality from Python?
And just convert to JS?

https://github.com/SmartBear/visualtest-python-sdk/blob/b7dc39c33ef837ba09b713b26895b1aec16f535c/sbvt/browser.py#L1031C5-L1031C5

Reply:
So there is WebDriverWait in cypress. That what I was asking before because I couldn't find a way to do good wait.

Reply:
I couldn't find any solution that will convert functionality from python. Cypress just cant wait for JS scripts check.